### DISCUSSION TITLE
Hitman 3 gives "Epic validation failed." when running with --offline

### DISCUSSION THREAD

**Discussion by thepwrtank18** [legendary-gl/legendary#304] [General]
![image](https://user-images.githubusercontent.com/57580668/125247889-0bd0ea00-e2b9-11eb-99b1-0cbe593b066f.png)

**Comment by thepwrtank18** [comment]
I noticed that a workaround was to make a batch file to HITMAN3.exe (not Launcher.exe) that has the arguments of `-EpicPortal -epicenv=Prod -epicusername=<your user name> -epicuserid=<your user id>`.

**Comment by derrod** [comment]
Yeah I don't think the game really supports any offline mode officially.
The next version of legendary will have an option to override the launch executable so you could bypass the launcher if necessary.

**Reply by thepwrtank18** [reply]
There is an offline mode in Hitman 3 (a bad one at that). It only actually gives you the option if:
* You declined the Privacy Policy
* Servers are down
* Authentication failed (e.g. pirated copy or Epic Games servers down *when game is already running*)
* You launched via the workaround
